homeassistant/components/lidarr/sensor.py

```diff
@@ -0,0 +1,161 @@
+"""Support for Lidarr."""
+from __future__ import annotations
+
+from datetime import datetime
+from typing import cast
+
+from aiopyarr import Diskspace, LidarrQueueItem
+from aiopyarr.models.lidarr import LidarrAlbum, LidarrCalendar
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DATA_GIGABYTES
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import LidarrEntity
+from .const import BYTE_SIZES, DOMAIN
+from .coordinator import LidarrDataUpdateCoordinator
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="commands",
+        name="Commands",
+        native_unit_of_measurement="Commands",
+        icon="mdi:code-braces",
+    ),
+    SensorEntityDescription(
+        key="diskspace",
+        name="Disk Space",
+        native_unit_of_measurement=DATA_GIGABYTES,
+        icon="mdi:harddisk",
+    ),
+    SensorEntityDescription(
+        key="queue",
+        name="Queue",
+        native_unit_of_measurement="Albums",
+        icon="mdi:music",
+    ),
+    SensorEntityDescription(
+        key="status",
+        name="Status",
+        native_unit_of_measurement="Status",
+        icon="mdi:information",
+    ),
+    SensorEntityDescription(
+        key="wanted",
+        name="Wanted",
+        native_unit_of_measurement="Albums",
+        icon="mdi:music",
+    ),
+    SensorEntityDescription(
+        key="upcoming",
+        name="Upcoming",
+        native_unit_of_measurement="Albums",
+        icon="mdi:music",
+    ),
+)
+
+PARALLEL_UPDATES = 1
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Lidarr sensors based on a config entry."""
+    async_add_entities(
+        LidarrSensor(hass.data[DOMAIN][entry.entry_id], description)
+        for description in SENSOR_TYPES
+    )
+
+
+class LidarrSensor(LidarrEntity, SensorEntity):
+    """Implementation of the Lidarr sensor."""
+
+    coordinator: LidarrDataUpdateCoordinator
+
+    def __init__(
+        self,
+        coordinator: LidarrDataUpdateCoordinator,
+        description: SensorEntityDescription,
+    ) -> None:
+        """Create Lidarr entity."""
+        super().__init__(coordinator)
+        self.entity_description = description
+        self._attr_name = f"Lidarr {description.name}"
+        self._attr_unique_id = f"{coordinator.config_entry.entry_id}/{description.name}"
```

Comment:
```suggestion
        self._attr_unique_id = f"{coordinator.config_entry.entry_id}/{description.key}"
```

Perhaps use the key here? I assume the keys won't change, where perhaps the name will change in the future.